grpc/server/src/engine_server/mod.rs

```diff
@@ -451,7 +451,11 @@ where
         };
 
         let response = match self.commit_step(correlation_id, request) {
-            Ok(StepResult::Success { post_state_hash }) => {
+            // TODO: Update protobufs for StepResult to use GetEraValidatorsError and have the
+            // validator weights
```

Comment:
@henrytill @EdHastingsCasperLabs including `next_era_validators` in the protocol buffers drastically changes their format.  Not including them doesn't seem to break any tests... what should I expect to break here?

Reply:
I've updated the protocol buffers in https://github.com/CasperLabs/casper-node/pull/717/commits/67140cb59f3dcf950bb8a6402196ee6d72f03341